spec/decorators/school_decorator_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe PersonaDecorator do
```

Comment:
```suggestion
RSpec.describe SchoolDecorator do
```

Reply:
Resolved in [URL improvements](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/bcf290bc40d8af1d1efd2ba3013b830fa3160207)